packages/core/src/components/context-menu/contextMenuTarget.tsx

```diff
@@ -6,9 +6,11 @@
  */
 
 import * as React from "react";
+import * as ReactDOM from "react-dom";
 
 import { CONTEXTMENU_WARN_DECORATOR_NO_METHOD } from "../../common/errors";
 import { isFunction, safeInvoke } from "../../common/utils";
+import { isDarkTheme } from "../../common/utils/isDarkTheme";
```

Comment:
Then change this to `import * as ThemeUtils from "../../common/utils/themeUtils"`

And reference the function as `ThemeUtils.isDarkTheme(...)`.

Reply:
disagree, let's stick with the individual files in `utils/` directory